### DISCUSSION TITLE
Export JKS

### DISCUSSION THREAD

**Discussion by fearnie85** [Venafi/venafips#40] [Q&A] [answered]
Hi all,

I am a little lost, I have exported a certificate in JKS format... but it is in Base64 format. I am curious how I now convert this to JKS format?
I tried to decode using PS 
[System.Text.Encoding]::Ascii.GetString([System.Convert]::FromBase64String($Certificate.CertificateData))

But I am really unsure what I should do with this.
Any help is appreciated it. Thanks.

**Comment by gdbarron** [comment, marked as answer]
This should work for you...
```powershell
$filename = 'C:\path\to\file'

$bytes = [Convert]::FromBase64String($Certificate.CertificateData)
[IO.File]::WriteAllBytes($filename, $bytes)
```

**Reply by fearnie85** [reply]
Great thanks :)

**Reply by gdbarron** [reply]
Can you please mark as the answer if this worked? :)

**Reply by fearnie85** [reply]
sorry! I overlooked this :) sure.

**Reply by fearnie85** [reply]
ah I see you did it. sorry and thanks for your help!
